unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -140,7 +140,7 @@ const filters: Filter[] = [
   {
     key: 'expiration',
     label: 'Expiration',
-    options: [MemberFilters.ACTIVE, MemberFilters.EXPIRED, MemberFilters.ALL],
+    options: ['active', 'expired', 'all'],
```

Comment:
you can actually type this btw. 

```ts
type Filters = "Active" | "Expired" | "All"

// use it elsewhere

options: Filters[]
option: Filter

```

Reply:
oh, i do it by purpose, but eventually can be changed when we need different filters!